The Chemistry and Brigmedic Update (+engi & robo tweaks)

wonderfulnewworld opened the pull request:
## Short description
<!-- What do you propose to change with your PR? -->
Adds a lot of cosmetic senior outfits/variant accessories to Chemist and Brigmedic, alongside corresponding PDAs, a Corpsman variant PDA for Brigmedic, and expands both of their character creation options. In addition to these cosmetic changes, it also adds 3 new beakers. The light and dark beakers are available to Chemists/Brigmedics respectively, as loadout options. They are 100u, unbreakable, **sealable** beakers. At 15 hours of the respective role, the silver beaker is made available as loadout. It's 150u, and a good quality of life for both roles.

Likewise, several silvers have been added to medical's vendor, priced at 200 credits. The golden beaker's price has been increased to 300 credits, but a third one has been added to the dispenser, so that brigmedic can no longer rob a chemist of the ability to mix chems faster.

The brigmedic's two new overcoats have altered stats when compared to their other 2 outer clothings. They have lower pierce/blunt resist, but higher slash resist, and infection resistance.

A few other changes I was working on made it into this PR, and as such, I will list them here:
- Roboticist Bags have been added to their drobe. (In preparation for a future Master Roboticist update.)
- Added the flipped black cap to the Roboticist's loadout.
- Added the Industrial and Advanced Industrial Welders to the engineering dispenser, at 250/400 credits respectively.
- Added a Golden Engineering Hardsuit to parallel the Golden Atmos Hardsuit.
- ~~The golden hardsuits have hade their prices increased to 1500 credits, but are now 15% lighter. Their armor has been reduced, making them a fast, but less durable variant of their base suits.~~
- The Skirt of Life is now available to Brigmedic as loadout, since it's always in their locker anyways. A Suit of Life and Nurse Hat of Life have also been added. The Skirt/Suit of Life have 5% infection resistance.
- The Ice Bucket is now a 2x2 item instead of 2x1, so it can no longer be used as a free golden beaker. It's a bucket.
- Added a Blueshield Officer eyepatch hud. We already had the other 2. It's available as a loadout option.

## Why we need to add this
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->
Most of the impactful Chemistry/Brigmedic changes come from the Chemistry/Brigmed QoL suggestions thread: https://discord.com/channels/1272545509562777621/1433188274549297214

The Roboticist changes were suggested by Lex. (Cvlancer)

The Golden hardsuit changes were a response to my disappointment at finding out that despite costing 1140 credits prior to this change, they had no actual functional differences. Now they're a speed-variant, but aren't as resistive.

## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->
<img width="499" height="533" alt="image" src="https://github.com/user-attachments/assets/49d0e604-8c98-47e3-b100-486d57dc7091" /> 

(I fixed the PDAs after taking this picture, they animate correctly now. The white + on the Corpsman PDAs was changed to black, too.)
<img width="776" height="543" alt="image" src="https://github.com/user-attachments/assets/ea891135-e83c-43d8-9f58-5a014c17c12a" />

<img width="784" height="693" alt="image" src="https://github.com/user-attachments/assets/40da2f35-3e68-4b88-86f6-bd3a11780d77" />

<img width="781" height="489" alt="image" src="https://github.com/user-attachments/assets/00f64768-3e21-43cc-95a6-9977695e990a" />

<img width="786" height="307" alt="image" src="https://github.com/user-attachments/assets/0f43cb97-6ce4-4823-b0fd-5bce9cba4545" />

<img width="785" height="340" alt="image" src="https://github.com/user-attachments/assets/38d20033-a4e1-492d-8010-3753e0acd99e" />

<img width="787" height="370" alt="image" src="https://github.com/user-attachments/assets/62c8f8f2-1919-4bab-85be-4f0957c0ca37" />

<img width="795" height="338" alt="image" src="https://github.com/user-attachments/assets/60933312-695d-4fc7-94cf-39aeafa109c9" />

<img width="793" height="339" alt="image" src="https://github.com/user-attachments/assets/3f253c8a-18d7-40cb-8b4d-7d5d98d5e28a" />

<img width="798" height="221" alt="image" src="https://github.com/user-attachments/assets/c1de5cc3-be95-465a-aebc-d423c869f8bd" />

<img width="782" height="402" alt="image" src="https://github.com/user-attachments/assets/7eb3eb31-d5ae-4408-b369-d3a226720d18" />

<img width="781" height="472" alt="image" src="https://github.com/user-attachments/assets/d020e0ec-0e5f-446b-bd91-0b65f5674041" />

<img width="783" height="302" alt="image" src="https://github.com/user-attachments/assets/a11d0bd6-3bca-43d1-b2aa-35bb5b613fdf" />

<img width="787" height="334" alt="image" src="https://github.com/user-attachments/assets/f399ab90-4245-4be8-a09f-8e83d9a2b507" />

<img width="792" height="362" alt="image" src="https://github.com/user-attachments/assets/cac979cd-1e9f-49f7-ada4-6f261f88d53e" />

<img width="780" height="266" alt="image" src="https://github.com/user-attachments/assets/cdea70a9-a0b5-4d47-a31b-bb99ee2bf940" />

<img width="930" height="460" alt="image" src="https://github.com/user-attachments/assets/4785d2ec-7a94-4517-96e0-499959d6ccb5" />

<img width="147" height="134" alt="image" src="https://github.com/user-attachments/assets/ea77e6a3-1300-4720-8fdd-1321d073c854" />

There are no ways to get additional light/dark beakers mid-round, so, as such, I think they are fine being 2x1.

<img width="571" height="266" alt="image" src="https://github.com/user-attachments/assets/99a1a170-9d3b-4062-8f56-9085fbe24935" />

The dark beaker is security contraband... because of this next image.

<img width="308" height="142" alt="image" src="https://github.com/user-attachments/assets/d4977317-ddd5-4eef-b653-a9fe54fa29c2" />

<img width="316" height="142" alt="image" src="https://github.com/user-attachments/assets/c59b4317-d7b8-4f8c-94d5-7e27ca7d1407" />

The light beaker, on the other hand, is not.

<img width="360" height="480" alt="image" src="https://github.com/user-attachments/assets/6f4d8fe4-f15f-47e5-8f64-9b713bc0fcc3" />

~~Considering lowering the armor even further, and increasing the price if this is still too good. I think the speed is the main appeal of this, and would like to keep it. (It has the same speed as the luxury mining hardsuit.)~~ Golden Hardsuits no longer have any stat differences, and cannot until there is a way to make hardsuits.

<img width="882" height="706" alt="image" src="https://github.com/user-attachments/assets/3a1880a9-f265-4a75-b95c-4c0921aeb02f" />

<img width="504" height="610" alt="image" src="https://github.com/user-attachments/assets/ed4a47ab-3cb2-48c6-a0cd-52acfe55cd84" />

<img width="394" height="572" alt="image" src="https://github.com/user-attachments/assets/8ea21eb0-e45b-4bf8-9679-a82d039636db" />

<img width="395" height="712" alt="image" src="https://github.com/user-attachments/assets/85211912-01ce-4fdc-9984-364e8dbe96db" />

The remaining Chemistry attire not shown in the drobe is locked behind Skilled/Senior Chemist time respectively. (15/60 hours. 60 hours has been chosen for parity with Veteran Mining.)

<img width="383" height="620" alt="image" src="https://github.com/user-attachments/assets/fbeaefee-007c-4713-89e1-3feb6b3abc2b" />

The Suit of Life and Nurse Hat of Life have been added to the manager wire medidrobe, since the Skirt of Life was already in there.



## Checks
<!-- check boxes for faster reviewing of your PR -->

- [X] I do not require assistance to complete the PR.
- [X] Before posting/requesting review of a PR, I have verified that the changes work.
- [X] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [X] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
:cl: wonderfulnewworld
- add: Senior Chemist Jumpsuit/Jumpskirt, Senior Chemist Lab Coat, Senior Chemist Winter Coat, Chemistry Trucker Hat, Chemistry Fishing Cap, Senior Chemistry Bag, Senior Chemist PDA (60 Hours Chemist)
- add: Advanced (Chemical) Jensens, Chemical HUD, Chemical HUD Eyepatch, Silver Beaker (15 hours Chemist)
- add: Chemistry beret, Chemistry nurse hat, light beaker, chemistry winter boots
- add: Golden Hand Labeler (100 Hours Chemist)
- add: Veteran Corpsman Response Jacket, Veteran Corpsman Trauma Wear, Corpsman's Nostalgic Overcoat, Corpsman's Traumabay hat, Veteran Brigmedic/Veteran Corpsman PDA (60 Hours Brigmedic)
- add: Brigmedic HUD eyepatch, Suit of Life, Nurse Hat of Life, Silver Beaker (15 Hours Brigmedic)
- add: Brigmedic's Reminisced Overcoat, Brigmedic's Medbay hat, dark beaker
- add: BlueShield Officer's HUD eyepatch
- tweak: Added Skirt of Life to Brigmedic's loadout
- tweak: Brigmedic can now select loadout options for: Mask, tie, ID, and beaker
- tweak: Chemist can now select loadout options for: Head, bag, ID, eyewear, beaker, and labeler
- tweak: The Ice Bucket is now 2x2
- tweak: Added 5 Silver Beakers to the Medical Dispenser, priced at 200 credits each
- tweak: Added an extra Golden Beaker to the Medical Dispenser
- tweak: Golden Beakers now cost 300 credits
- tweak: Added the Industrial Welder/Advanced Industrial Welder to the Engineering Dispenser (at 250/400 credits respectively)
- add: added a Golden Engineering Hardsuit to the Engineering Dispenser
- tweak: Added the flipped black cap to the Roboticist's loadout
- tweak: Added the Roboticist's bags to the RoboDrobe
- tweak: The Skirt of Life/Suit of Life now give 5% Infection resist
- tweak: Added the Suit of Life and Nurse Hat of Life to the MediDrobe's manager wire
- tweak: Added the Chemistry beret, Chemistry nursehat, and Chemistry winter boots to the ChemDrobe
- tweak: Brigmedic now has Surgery access (it was forgotten)

wonderfulnewworld committed:
Chemist and Brigmed senior PDAs and special Beakers

Fighting over the golden beaker no more

wonderfulnewworld committed:
Fix beaker transparency and add damage to beakers

Light and dark beakers finished

wonderfulnewworld committed:
Renamed filenames of senior brigmedic, and fixed double PDA bug

Feature complete

wonderfulnewworld committed:
Indent fix on light beaker

Intented too far

wonderfulnewworld committed:
Fix veteran corpsman pda in meta.json

Forgot to change it

wonderfulnewworld committed:
Senior chemist outfit textures

I guess I'm just committing to this at this point

wonderfulnewworld committed:
Senior Chemist labcoat

I guess I'm doing all of this now

wonderfulnewworld committed:
Advanced jensens

They're chemgoggles

wonderfulnewworld committed:
ChemHUD

Now to work on Brigmedic

wonderfulnewworld committed:
"Things only get added when 1 guy has an autistic obsession"

I realize now that I have become that guy.

wonderfulnewworld committed:
Added a lot of cosmetics

Chem/brigmedic clothes, hats, and some eyepatches (+bso eyepatch)

wonderfulnewworld committed:
Chemist pda is now pink + corpsman pda

Might as well commit to the distinction

wonderfulnewworld committed:
The Golden Labeler

People will lose their heads over this...

wonderfulnewworld committed:
Gold labeler is now named correctly

A name deserving of such a feat.

wonderfulnewworld committed:
I like this name better

golden labeler

wonderfulnewworld committed:
Chemistry file structure part 1

Going to bed, will finish pr tomorrow... oh it's 3 am.

wonderfulnewworld committed:
Brigmed and Blueshield loadouts

Just need to implement them, now.

wonderfulnewworld committed:
Brigmedic now has surgery access

They were missing it.

wonderfulnewworld committed:
Implemented prototypes & vending tweaks

Golden Atmos Hardsuit is now 10% faster

wonderfulnewworld committed:
Golden Engi hardsuit

Both the engi and atmos ones cost 1500 now, but are 10% faster to make up for it.

wonderfulnewworld committed:
Screw it, 15% instead

They cost 1500.

wonderfulnewworld committed:
Senior chemist winter clothes

Slight golden hardsuit nerf too

wonderfulnewworld committed:
I added the flipped black cap to roboticist

Enjoy, Lex. (Also roboticist bags.)

wonderfulnewworld committed:
added chem stuff to chemdrobe

And fixed roboticist

wonderfulnewworld committed:
Fixed brigmed PDAs + nerfed golden hardsuits

Also added jumpsuit of life/nurse hat of life to medidrobe + Fixed chemist typo.

wonderfulnewworld committed:
Slight recoloring on corpsman PDAs

It bothered me.

wonderfulnewworld committed:
final renames

feature complete

wonderfulnewworld committed:
ice bucket is now a normal size

200u 2x1 unbreakable container, nerf was a suggestion by Dopple.

wonderfulnewworld committed:
Dark beaker is now sec contra

Harmbeaker

Fasuh commented:
I believe the idea behind items that are available in the vends what that they should be at most as good as options available to the crew who are not paying at the vend. 
I say that mostly at the golden engineering hardsuits, when i brought up the question about "why are we adding them if they are not any different" i got an answer that we dont want to put items that are strictly better in the vends. 

I say this mostly out of concern that those hardsuits probably do not meet this requirement, as armor is really useless to engineers and atmos technicians, and 15% more speed is an enormous buff to their usability.

StarlightHost commented:
The comment above is almost correct.

The restriction on vending machines for credits works like this.
1. If there is a starting tier and a science tier that is significantly better, then the credit option sits between them. For example, in medicine there is T1 starter medical gear, T2 medical gear for credits, and T3 medical gear unlocked through science.

2. If there is no science tier, like with hardsuits, then the item stats are equivalent to T1, but with a resprite. Stunbatons are an example.

There are two options here. Either the hardsuit needs to be reverted back, or a science tier hardsuit needs to be created that is even better, but can be unlocked by anyone.

wonderfulnewworld committed:
Removed golden hardsuit stat differences

No current way to make hardsuits means science cannot research a better one

wonderfulnewworld commented:
For now, and for this PR as it stands, I've removed the stat differences of the golden hardsuits, reverting them back to how they were before, as there's currently no way to *make* hardsuits in order to adhere to point 2 (outside of the clown/mime hardsuits), and trying to solve that is outside the scope of the PR. I guess in theory they could be added to the exofab though, could be a neat thing to tackle in the future.

wonderfulnewworld commented:
I have fabricated an idea to allow for hardsuit fabrication, and will make a separate PR for it. In that PR, I'll re-buff the golden hardsuits, since there will be a better hardsuit that can be made by science.

redboit commented:
nondescript advanced jensen glasses lol

redboit commented:
/srs we need more jensens.

Conflee commented:
The chemistry winter coat and winter boots feel very bright, otherwise stuff looks nice

wonderfulnewworld commented:
I took the exact med winter coats values and just shifted the H of the HSV over to orange, but I can make them darker if need be. Wanted to keep them the saturation and value consistent.

wonderfulnewworld committed:
Merge branch 'starlight-dev' into mo-beakers-and-extras

wonderfulnewworld commented:
Fixed a merge conflict, not the color change.

wonderfulnewworld committed:
Made chem winter gear (except hood) darker

The hood was already darker. It's a bit easier to see the definition now.

wonderfulnewworld committed:
Merge branch 'mo-beakers-and-extras' of https://github.com/wonderfulnewworld/luminous-skylight-station into mo-beakers-and-extras

wonderfulnewworld commented:
<img width="985" height="506" alt="image" src="https://github.com/user-attachments/assets/7cda43e0-32d0-48e3-9b6c-c0c0a454fe9e" />
Chemist winter gear is darker now.

wonderfulnewworld committed:
Might as well do the hood too

This should be complete now.

wonderfulnewworld committed:
fixed soundHit on the beakers

it was failing a test

wonderfulnewworld commented:
it finally passed all checks

CrazyPhantom779 commented on Resources/Prototypes/Catalog/VendingMachines/Inventories/chemdrobe.yml:
```diff
@@ -11,9 +11,12 @@
     ChemBag: 2
     HandLabeler: 3
     ClothingBeltMedical: 2
+    ClothingHeadHatBeretChemist: 2
```
Starlight comments

CrazyPhantom779 commented on Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml:
```diff
@@ -265,6 +265,40 @@
+  id: ClothingOuterWinterChemSenior
+  name: senior chemist winter coat
+  components:
+  - type: Sprite
```
move this to _starlight

CrazyPhantom779 commented on Resources/Prototypes/Entities/Clothing/Head/hoods.yml:
```diff
@@ -339,6 +339,19 @@
+  id: ClothingHeadHatHoodWinterChemSenior
+  categories: [ HideSpawnMenu ]
+  name: senior chemist winter coat hood
+  components:
```
move to _starlight

CrazyPhantom779 commented on Resources/Prototypes/Entities/Clothing/Shoes/boots.yml:
```diff
@@ -140,6 +140,18 @@
+- type: entity
+  parent: ClothingShoesBaseWinterBoots
+  id: ClothingShoesBootsWinterChem
+  name: chemistry winter boots
```
move this to _starlight

CrazyPhantom779 commented on Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_special.yml:
```diff
@@ -33,6 +33,8 @@
   - type: Sprite
     sprite: Objects/Consumable/Drinks/icebucket.rsi
     state: icon
+  - type: Item
```
comments

CrazyPhantom779 commented on Resources/Prototypes/Entities/Objects/Devices/pda.yml:
```diff
@@ -795,7 +795,7 @@
   parent: BaseMedicalPDA
   id: ChemistryPDA
-  name: chemistry PDA
+  name: chemist PDA
```
comments

CrazyPhantom779 commented on Resources/Textures/_Starlight/Clothing/Head/Hoods/Coat/hoodchemistsenior.rsi/meta.json:
```diff
@@ -0,0 +1,18 @@
+        "y": 32
+    },
+    "states": [
+        {
```
move to _starlight

CrazyPhantom779 commented on Resources/Textures/Clothing/OuterClothing/WinterCoats/coat.rsi/meta.json:
```diff
@@ -581,6 +581,21 @@
+    },
+    {
+      "name": "CHEM-SENIOR-equipped-OUTERCLOTHING",
+      "directions": 4
```
starlight comments around the states or move them to _starlight

CrazyPhantom779 commented on Resources/Textures/_Starlight/Clothing/Shoes/Boots/winterbootschem.rsi/meta.json:
```diff
@@ -0,0 +1,26 @@
+    {
+      "name": "icon"
+    },
+    {
```
move thsi to _starlight

CrazyPhantom779 reviewed (changes requested):
I think it would be better to redirect to _starlight instead of swapping out sprites but thats up to conf if there fine with it. Otherwise comments and move new entity/files to _starlight

wonderfulnewworld committed:
_Starlight migrations and # Starlight comments

Moved all the senior chemist stuff to _Starlight, added # Starlight comments to chemdrobe and ice bucket.

wonderfulnewworld commented:
Migrated the senior chem stuff to _Starlight and added the missing # Starlight comments to the chemdrobe and ice bucket.

CrazyPhantom779 commented on Resources/Textures/Clothing/OuterClothing/WinterCoats/coat.rsi/meta.json:
```diff
@@ -1,7 +1,7 @@
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/77cff42b6c514e73881a885036be4b4dd2949f62 recolored by Github user Dutch-VanDerLinde, CAP edited by Skarletto (github), equipped-OUTERCLOTHING-vox states taken from /vg/station at commit https://github.com/vgstation-coders/vgstation13/commit/31d6576ba8102135d058ef49c3cb6ecbe8db8a79 and repaletted to match  ...
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/77cff42b6c514e73881a885036be4b4dd2949f62 recolored by Github user Dutch-VanDerLinde, CAP edited by Skarletto (github), equipped-OUTERCLOTHING-vox states taken from /vg/station at commit https://github.com/vgstation-coders/vgstation13/commit/31d6576ba8102135d058ef49c3cb6ecbe8db8a79 and repaletted to match  ...
```
remove ``CHEM SENIOR edited by wonderfulnewworld (github).``

CrazyPhantom779 reviewed (changes requested):
You made changes to ``Resources/Textures/Clothing/Head/Hoods/Coat/hoodchemist.rsi/equipped-HELMET.png`` either redirect whatevers using it to a _starlight png or reference it in the meta

wonderfulnewworld committed:
Restored original chemist hood and fixed meta.json

I changed the hood on accident once and forgot to fix it.

CrazyPhantom779 reviewed (approved):
Im not so sure about adding the golden suits to the vending machines but its whatever, changes look good but theres alot to go over

wonderfulnewworld commented:
The golden atmos suit was already in there, I just made an engi one.

Conflee reviewed (approved):
Adjusted sprites good

walksanatora reviewed (approved):
```diff
+[CentCom]
+[Approved]
```

deltaVeIocity commented:
My ice bucket... I'm gonna die of thirst now
